Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AtomRenderPlugin.cpp

```diff
@@ -41,13 +41,14 @@ namespace EMStudio
 
     AtomRenderPlugin::~AtomRenderPlugin()
     {
+        AzToolsFramework::ViewportInteraction::ViewportMouseRequestBus::Handler::BusDisconnect();
+        delete m_animViewportWidget;
```

Comment:
I don't 100% understand the details, but I found I had to add this to avoid a crash when you close the animation editor (only occurring with my code added). I think qt does a certain amount of autodeletion, which might be why this wasn't needed before, but maybe it wasn't happening at the correct time.